src/site/navigation/sidebar_nav.drupal.liquid

```diff
@@ -35,7 +35,7 @@
                                         {% for link in link.links %}
                                             <li {% if link.url.path contains entityUrl.path %} class="active-level" {% endif %}>
                                                 <a {% if entityUrl.path == link.url.path %} class="usa-current" {% endif %}
-                                                        href="{{ link.url.path }}" onClick="recordEvent({ event: 'nav-sidenav' });">{{ link.label }}</a>
+                                                    lang="{{link.url.path | detectLang }}"   href="{{ link.url.path }}" onClick="recordEvent({ event: 'nav-sidenav' });">{{ link.label }}</a>
```

Comment:
Could you add a unit test for `detectLang`'s liquid filter here if you have time? 🙂  https://github.com/department-of-veterans-affairs/vets-website/blob/0dd1d86a167f61b31720a42803647cd675b24e61/src/site/filters/liquid.unit.spec.js

Reply:
done!